common/web/gesture-recognizer/src/engine/headless/simpleGestureSource.ts

```diff
@@ -96,6 +100,18 @@ export class SimpleGestureSource<HoveredItemType> {
     return this.path.coords[this.path.coords.length-1];
   }
 
+  public constructSubview(startAtEnd: boolean, preserveBaseItem: boolean) {
+    return new SimpleGestureSourceSubview(this, startAtEnd, preserveBaseItem);
+  }
+
+  public terminate(cancel?: boolean) {
+    this.path.terminate(cancel);
+  }
+
+  public get isPathComplete(): boolean {
```

Comment:
Should we have doc comments on these? Especially given things that might not be obvious, like what does `cancel` mean for `terminate`?